cpp_guidelines/.clang-format

```diff
@@ -0,0 +1,107 @@
+---
+Language: Cpp
+Standard: c++17
+DisableFormat:   false
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
```

Comment:
**VOTE**
current | false |
--------|-----------|
 0      |   1 |

Aligning assignments can cause unnecessarily large diffs and the formatting might turn out really bad sometimes.

Reply:
current | false
-- | --
0 | 2


Current (true):
```C++
        template <
            typename T_MplSeq,
            typename T_TmpResult = bmpl::vector0<>,
            bool T_isEmpty       = bmpl::empty<T_MplSeq>::value>
        struct AllCombinations;
```

Suggestion (false):
```C++
        template <
            typename T_MplSeq,
            typename T_TmpResult = bmpl::vector0<>,
            bool T_isEmpty = bmpl::empty<T_MplSeq>::value>
        struct AllCombinations;
```

Reply:
current | false       |
--------|-------------|
 0      |   2        |

Reply:
Props to @psychocoderHPC for finding and example where `true` actually looks better and still voting false based on principle.

Reply:
current | false
-- | --
0 | 4

Reply:
current | false
-- | --
0 | 4

Reply:
current | false |
--------|-----------|
 0      |   6

Reply:
current | false
-- | --
0 | 7

Reply:
current | false       |
--------|-------------|
 0      |   8         |

Reply:
current | false |
--------|-----------|
 1      |   8

Reply:
current | false
-- | --
2 | 8

Reply:
current | false
-- | --
2 | 9

Reply:
Reminder: Voting will be closed on 2020-09-26 at 12:37 HZDR time.

Reply:
current | false |
--------|-----------|
 3      |   9

Reply:
Voting closed, the winner is `false`.